src/plugin.validation.dns.simply/Simply/SimplyDnsClient.cs

```diff
@@ -0,0 +1,100 @@
+﻿using PKISharp.WACS.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace PKISharp.WACS.Plugins.ValidationPlugins.Simply
+{
+    public class SimplyDnsClient
+    {
+        private readonly ILogService _log;
+        private readonly HttpClient _httpClient = new();

```

Comment:
Please get an HttpClient from the `IProxyService` so that it has the right proxy settings, logging, user agent settings, etc.